uwh-overlay-drawing/src/pages.rs

```diff
@@ -0,0 +1,402 @@
+                DrawTextureParams {
+                    dest_size: Some(vec2(100f32, 100f32)),
+                    ..Default::default()
+                },
+            );
+            let min = state.snapshot.secs_in_period / 60;
+            let secs = state.snapshot.secs_in_period % 60;
+            draw_text_ex(
+                format!("{}:{}", min, secs).as_str(),
+                923f32,
+                1020f32,
+                TextParams {
+                    font: self.textures.font(),
+                    font_size: 50,
+                    ..Default::default()
+                },
+            );
+            draw_text_ex(
+                "NEXT GAME",
+                905f32,
+                1044f32,
+                TextParams {
+                    font: self.textures.font(),
+                    font_size: 20,
+                    ..Default::default()
+                },
+            );
+        }
+    }
+
+    /// Display final scores after game is done
+    pub fn final_scores(&mut self, state: &State) {
+        draw_texture(*self.textures.atlantis_logo_graphic(), 0_f32, 0f32, WHITE);
+        draw_texture(*self.textures.final_score_graphic(), 0_f32, 0f32, WHITE);
+        draw_texture(
+            *self.textures.team_information_graphic(),
+            0_f32,
+            0f32,
+            WHITE,
+        );
+        draw_text_ex(
+            state.white.to_uppercase().as_str(),
+            340f32,
+            805f32,
+            TextParams {
+                font: self.textures.font(),
+                font_size: 50,
+                ..Default::default()
+            },
+        );
+        draw_text_ex(
+            state.black.to_uppercase().as_str(),
+            1240f32,
+            805f32,
+            TextParams {
+                font: self.textures.font(),
+                font_size: 45,
+                ..Default::default()
+            },
+        );
+        draw_text_ex(
+            state.snapshot.b_score.to_string().as_str(),
+            1400f32,
+            580f32,
+            TextParams {
+                font: self.textures.font(),
+                font_size: 180,
+                ..Default::default()
+            },
+        );
+        if self.is_alpha_mode {
+            draw_text_ex(
+                state.snapshot.w_score.to_string().as_str(),
+                430f32,
+                580f32,
+                TextParams {
+                    font: self.textures.font(),
+                    font_size: 180,
+                    ..Default::default()
+                },
+            );
+        } else {
+            draw_text_ex(
+                state.snapshot.w_score.to_string().as_str(),
+                430f32,
+                580f32,
+                TextParams {
+                    font: self.textures.font(),
+                    font_size: 180,
+                    color: BLACK,
+                    ..Default::default()
+                },
+            );
+        }
```

Comment:
There's still a lot of duplication here, which could be reduced by
```suggestion
        draw_text_ex(
            state.snapshot.w_score.to_string().as_str(),
            430f32,
            580f32,
            TextParams {
                font: self.textures.font(),
                font_size: 180,
                color: if self.is_alpha_mode { WHITE } else { BLACK },
                ..Default::default()
            },
        );
```